server/svix-server/src/v1/utils/mod.rs

```diff
@@ -288,7 +311,14 @@ pub trait ModelOut: Clone {
         } else {
             IteratorDirection::Next
         };
-        list_response_inner(data, limit, ListOrdering::Descending, direction, true)
+        list_response_inner(
+            data,
+            limit,
+            ListOrdering::Descending,
+            direction,
+            true,
+            false,
```

Comment:
1. This is going out of control with this true, false, and a million params.
2. Luckily it's not necessarily. `newest_first` == order descending. You don't need another param, you already have it. If it's newest first, you just do descending order...

Reply:
I made `apply_pagination` and `list_response` generic again, they simply take the ordering as a parameter.